src/components/Calendar/Event/CreateEvent.js

```diff
@@ -87,38 +101,92 @@ export default function CreateEvent(props) {
+            <Stack direction="row" spacing={2} justifyContent="center">
+                <PrimaryButton2 width={'20vw'} colour={'#0072A8'} content="Course" value="course"
+                                onClick={handleTypeUpdate}/>
+                <PrimaryButton2 width={'20vw'} colour={'#8CC63E'} content="Study" value="study"
+                                onClick={handleTypeUpdate}/>
+                <PrimaryButton2 width={'20vw'} colour={'#DA3A16'} content="Workout" value="workout"
+                                onClick={handleTypeUpdate}/>
+                <PrimaryButton2 width={'20vw'} colour={'#DB0272'} content="Appointment" value="appointment"
+                                onClick={handleTypeUpdate}/>
+            </Stack>
+        </React.Fragment>
+    )
+
+    const course = (
+        <React.Fragment>
+            <div style={{paddingTop: '10px'}}>
+                <TextField required
+                    id="outlined-required"
+                           label="subject"
+                           defaultValue="EX: SOEN"
+                           size="small"
+                           onChange={(e) => {
+                               setEventData({...eventData, subject: e.target.value})}}
+                />
+                <TextField required
+                           id="outlined-required"
+                           label="catalog"
+                           defaultValue="EX: 385"
+                           size="small"
+                           onChange={(e) => {
+                               setEventData({...eventData, catalog: e.target.value})}}
+                />
+
+            </div>
+
+        </React.Fragment>
+    )
 
 
     return (
         <React.Fragment>
-            <form style={{ paddingLeft: '10px', paddingRight: '10px' }}>
-                <div style={{ paddingTop: '10px', paddingBottom: '10px' }}>{PageError}</div>
-
-                <div align='center' style={{ paddingTop: '16px', paddingBottom: '20px' }}>
-
-                <EventForm  eventState={eventData} eventStateSetter={setEventData}/>
-                   
-                 <FormControlLabel sx={{display: 'block'}} label="Recurrent" control={
-                    <Switch
-                        sx={{color: '#912338'}}
-                        checked={isRecurrent}
-                        onChange={handleIsRecurrentChange}
-                    />
-                }/>
-                <div>{ isRecurrent && recurrenceSelection }</div>
-                <div>{ buttons }</div>
+            <form style={{paddingLeft: '10px', paddingRight: '10px'}}>
+                <div style={{paddingTop: '10px', paddingBottom: '10px'}}>{PageError}</div>
+
+                <div align='center' style={{
+                    overflow: 'auto',
+                    paddingTop: '10px',
+                    width: '90vw',
+                    height: '70vh'
+                }}>
+                    <div style={{paddingBottom: '10px'}}>{categories}</div>
```

Comment:
Why are the categories (and course) divs placed above the event form and the recurrence selection?

Reply:
so that they appear before the fields as per my picture in the issue

Reply:
I see. Thanks for the clarification.